tests/Microsoft.Identity.Test.Unit.net45/CacheTests/CacheSerializationTests.cs

```diff
@@ -398,7 +398,7 @@ public void TestDictionarySerialization()
 
             var otherAccessor = new InMemoryTokenCacheAccessor();
             var s2 = new TokenCacheDictionarySerializer(otherAccessor);
-            s2.Deserialize(bytes);
+            s2.Deserialize(bytes, false);
```

Comment:
Please modify an existing unit test or write a new one, where the flag is set to true to cover that logic as well.